Jenkinsfile

```diff
@@ -38,7 +38,7 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@samirrav/DAOS-1316") _
```

Comment:
Now that daos-stack/pipeline-lib#55 has landed, don't forget to revert this.

Reply:
Yes reverted